.eslintrc.js

```diff
@@ -20,23 +20,17 @@ module.exports = {
         'no-restricted-imports': [
             'error',
             {
-                paths: [
-                    {
-                        name: 'react-native',
-                        importNames: ['useWindowDimensions'],
-                        message: 'Please use useWindowDimensions from src/hooks/useWindowDimensions instead',
-                    },
-                    {
-                        name: 'react-native',
-                        importNames: ['StatusBar'],
-                        message: 'Please use StatusBar from src/libs/StatusBar instead',
-                    },
-                    {
-                        name: 'react-native',
-                        importNames: ['TouchableOpacity', 'TouchableWithoutFeedback', 'TouchableNativeFeedback', 'TouchableHighlight', 'Pressable'],
-                        message: 'Please use PressableWithFeedback and/or PressableWithoutFeedback from src/components/Pressable instead',
-                    },
-                ],
+              paths: [
+                  {
+                    name: 'react-native',
+                    importNames: ['useWindowDimensions', 'StatusBar', 'TouchableOpacity', 'TouchableWithoutFeedback', 'TouchableNativeFeedback', 'TouchableHighlight'],
+                    message: `If you're using following components from react native:\n
```

Comment:
I think this format is more readable:
```
message: [
  "For 'useWindowDimensions', please use 'src/hooks/useWindowDimensions' instead.",
  "For 'TouchableOpacity', 'TouchableWithoutFeedback', 'TouchableNativeFeedback', 'TouchableHighlight', please use 'PressableWithFeedback' and/or 'PressableWithoutFeedback' from 'src/components/Pressable' instead.",
  "For 'StatusBar', please use 'src/libs/StatusBar' instead.",
].join('\n'),
```

Reply:
Hi @blazejkustra, I edited the message format as you suggested. But I think adding one more new line at top, makes it more readable. So, in VS code the final result seems like the following:
 
![Screenshot 2023-07-11 at 18 54 15](https://github.com/Expensify/App/assets/77512248/5748843f-072a-4e63-a71b-d2ecd15d0826)

Thank you for your advice.